packages/codewarsClassifier/serverless.yml

```diff
@@ -0,0 +1,117 @@
+service: karate-classifier
+
+plugins:
+  - serverless-pseudo-parameters
+  - serverless-webpack
+  - serverless-dynamodb-local
+  - serverless-offline
+
+provider:
+  name: aws
+  runtime: nodejs6.10
+  stage: test
+  region: eu-west-1
+  environment:
+    CODEWARS_ACCOUNTS_SHEET: ${env:CODEWARS_ACCOUNTS_SHEET}
+    SCORE_SHEET: ${env:SCORE_SHEET}
+    SENSEI_URL: ${env:SENSEI_URL}
+  memorySize: 512
+  timeout: 300
+
+# you can overwrite defaults here
```

Comment:
do we still need this commented out code here?

Reply:
yes, we are, because it will be very helpful for further configuration